Vagrantfile

```diff
@@ -100,6 +100,26 @@ Vagrant.configure("2") do |config|
     centos.vm.synced_folder ".", "/vagrant", type: "rsync"
   end
 
+  config.vm.define "x86_64-c10s" do |centos|
+    centos.vm.box = "centos-10-stream"
+    centos.vm.box_url = get_box("https://cloud.centos.org/centos/10-stream/x86_64/images/", /.*latest\.x86_64\.vagrant-libvirt\.box$/)
+
+    centos.vm.provision "shell", inline: <<-SHELL
+       dnf config-manager --set-enabled crb
```

Comment:
Why do we need CRB specifically for c10s? Can you add a comment?

Reply:
done